printer_mods/padok/chain_anchor-ADXL345_mount/README.md

```diff
@@ -0,0 +1,50 @@
+# Chain Anchor - ADXL345 mount Mod
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/VORON2_v2.4_Assembly_-_Chain_Anchor_-_IGUS.png)
```

Comment:
Please simplify all the image references, you don't need to go up two directories and then back down.  This image should be just

    ![Chain Anchor](img/VORON2_v2.4_Assembly_-_Chain_Anchor_-_IGUS.png)